source/code/plugins/filter_syslog.rb

```diff
@@ -30,6 +30,7 @@ def filter(tag, time, record)
       # Use Time.now, because it is the only way to get subsecond precision in version 0.12.
       # The time may be slightly in the future from the ingestion time.
       record["Timestamp"] = OMS::Common.format_time(Time.now.to_f)
+      record["EventTime"] = OMS::Common.format_time(time)
```

Comment:
That makes it clearer. Thanks for the investigation.

There's still a potential issue with syslog messages that are forwarded from another machine with a different time zone. It seems that the syslog timestamp is not adjusted to the receiving machine time zone in this case.
I sent you the details of the machine if you'd like to take a look.